packages/spacecat-shared-data-access/src/models/site/config.js

```diff
@@ -31,6 +31,7 @@ export const configSchema = Joi.object({
       Joi.object({
         disabled: Joi.boolean().optional(),
         excludedURLs: Joi.array().items(Joi.string()).optional(),
+        manualOverwrites: Joi.array().items(Joi.string()).optional(),
```

Comment:
This would not be an array of string, but rather an array of objects, with broken URL -> URL to overwrite it with.